deps.versions.toml

```diff
@@ -6,12 +6,12 @@ essenty = "1.2.0-alpha-03"
 parcelizeDarwin = "0.1.4"
 reaktive = "1.2.3"
 junit = "4.13.2"
-jetbrainsCompose = "1.4.0"
+jetbrainsCompose = "1.5.0-beta01"
```

Comment:
Compose Multiplatform [1.5.0-beta02](https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.5.0-beta02) is available. Maybe use that one?